### PR TITLE
doc: fix literalExpression in examples

### PR DESCRIPTION
**Pull request by MarcelCoding**
Before:
image

After:
image

### CODE REVIEW

File: nixos-modules/host/options.nix [hunk 1]
```diff
@@ -32,14 +32,13 @@
...
+      example = lib.literalExpression ''
         # Attach tap interface to bridge br0, and bring it up
-        ''${pkgs.iproute2}/bin/ip link set "$id" master br0 up
+        "${pkgs.iproute2}/bin/ip link set \"$id\" master br0 up"
       '';
       type = types.lines;
-      default = ''
-        ${pkgs.iproute2}/bin/ip link set "$id" up
-      '';
+      default = "${pkgs.iproute2}/bin/ip link set \"$id\" up";
```
Comment (astro) [inline nixos-modules/host/options.nix]: Can we keep the `default = ''` with a newline, now that we have the `defaultText` too?

File: nixos-modules/microvm/options.nix [hunk 2]
```diff
@@ -148,8 +148,7 @@ in
           };
         });
       default = [];
-      example = lib.literalExpression
-        ''
+      example = lib.literalExpression /* nix */ ''
```
Comment (astro) [inline nixos-modules/microvm/options.nix:151]: I like removing the linebreak here but what about the `/* nix */`? Not gonna enforce a `nixfmt` in this project! :laughing:

**Review by astro — CHANGES_REQUESTED** [review]

File: nixos-modules/microvm/options.nix (hunk 2 above)
Comment (MarcelCoding) [reply nixos-modules/microvm/options.nix:151]: It's tree sitter magic that allows tree sitter to inject a language in the string.

File: nixos-modules/host/options.nix [hunk 3]
```diff
@@ -32,14 +32,13 @@
...
+      defaultText = "${pkgs.iproute2}/bin/ip link set \"$id\" up";
```
Comment (SuperSandro2000) [inline nixos-modules/host/options.nix]:
```suggestion
      default = ''
        ${pkgs.iproute2}/bin/ip link set "$id" up
      '';
      defaultText = ''
        ${pkgs.iproute2}/bin/ip link set "$id" up
      '';
```

**Comment by astro** [comment]
Cheers!

**Pull request merged**